### ISSUE TITLE
Causes players to get kicked right after a user is done linking discord account to mc account.

### ISSUE THREAD

**Issue by frikinjay** [CmdrJane/Discordium#3]
I am assuming its discordium because it happens right after a a new user is done linking discord id to mc id using the code. The only reference there is to anything going wrong is below. Theres nothing else to go on.
```
[Render thread/ERROR]: Registry remapping failed!
net.fabricmc.fabric.impl.registry.sync.RemapException: Received ID map for minecraft:game_event contains IDs unknown to the receiver!
 - minecraft:click
	at net.minecraft.class_2370.remap(class_2370.java:694) ~[client-intermediary.jar:?]
	at net.fabricmc.fabric.impl.registry.sync.RegistrySyncManager.apply(RegistrySyncManager.java:314) ~[fabric-registry-sync-v0-0.9.0+befed49e65-f30e34e5da83b8b7.jar:?]
	at net.fabricmc.fabric.impl.registry.sync.RegistrySyncManager.lambda$receivePacket$0(RegistrySyncManager.java:125) ~[fabric-registry-sync-v0-0.9.0+befed49e65-f30e34e5da83b8b7.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) [?:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:151) [client-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) [client-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:125) [client-intermediary.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:110) [client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1080) [client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:733) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:238) [client-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.12.12.jar:?]
```

**Comment by CmdrJane** [comment]
you are using same mods as before?

**Comment by frikinjay** [comment]
Yes

**Comment by CmdrJane** [comment]
this happens after sending code to bot pm?

**Comment by frikinjay** [comment]
Yes, once the player is done with the code. The user that did it and other users that try to join after and have already done verification before get this error and disconnect when trying to join.

**Comment by CmdrJane** [comment]
if possible can you provide me your server files? (you can delete world folder i dont need it and don forget to delete bot token from discordium config)

**Comment by frikinjay** [comment]
Nvm turns out some other mod is responsible. Thanks for your time. I will close it.

**Comment by CmdrJane** [comment]
wait if you know the mod that caused it can you tell me which one it was?
